Checks/PWR081/README.md

```diff
@@ -0,0 +1,170 @@
+# PWR081: Conditionally initialized output arguments can lead to undefined behavior
```

Comment:
I wouldn't say "Conditionally". It could not be set at all and the potential UB would be still there. I'm aware that the compiler already would complain if the argument is not used at all